tests/test_tooltip.py

```diff
@@ -0,0 +1,30 @@
+import time
+from dash import Dash, _dash_renderer
+import dash_mantine_components as dmc
+
+_dash_renderer._set_react_version("18.2.0")
+
+
+def test_001to_tooltip(dash_duo):
+    app = Dash(__name__)
+
+    component = dmc.Tooltip(
+        dmc.Text("Text", id="text"),
+        label="Tooltip text",
+    )
+
+    app.layout = dmc.MantineProvider(component)
+
+    dash_duo.start_server(app)
+
+    # Wait for the app to load
+    dash_duo.wait_for_text_to_equal("#text", "Text")
+    dash_duo.find_element("#text").click()
+
+    time.sleep(.2)  # Adjust if needed based on any transition delays
```

Comment:
Do you really need the explicit delay? Is the `find_element` below not sufficient?

Reply:
It passed when I ran it locally, but failed here.  I'll put the delay back and see if it passes